src/chocolatey/infrastructure/filesystem/IFileSystem.cs

```diff
@@ -188,6 +188,14 @@ public interface IFileSystem
         /// <param name="newFilePath">The new path for the file. </param>
         void MoveFile(string filePath, string newFilePath);
 
+        /// <summary>
+        ///   Moves a specified file to a new location, providing the option to specify a new file name.

```

Comment:
Do we need to extend this regarding the ability to make it silent?

Reply:
I don't think this is needed, as it is mentioned as the documentation in the parameters.

Reply:
The reason I asked the question, both here and in the other methods, is that in the old method that now has an overload, the summary is :

>  Moves a specified file to a new location, providing the option to specify a new file name.

And there are param's for both `filePath` and `newFilePath`.  We have added a new overload with a new parameter, where all we have done is add a new `param`.  Given that both the existing parameters are called out in the summary previously, I thought it made sense to include mention of the new parameter.

Happy for this to not be included, since as you mention, it is covered in the parameter description, but then that brings up, why the original summary wasn't simply:

> Moves a specified file to a new location.